src/components/cc-feature-list/cc-feature-list.js

```diff
@@ -0,0 +1,212 @@
+   * @return {TemplateResult}
+   * @private
+   */
+  _renderFeature(feature) {
+    let featureStatus = this._getFeatureStatus(feature.status);
+    return html`
+      <div class="feature">
+        <div class="header">
+          <div class="header-title">
+            <div class="title">${feature.name}</div>
+            <div class="status">
+              ${feature.status
+                ? html` <cc-badge intent="${featureStatus.intent}"> ${featureStatus.text} </cc-badge> `
+                : ''}
+            </div>
+          </div>
+          <cc-toggle
+            .choices="${feature.options}"
+            value="${feature.value}"
+            @cc-select=${(/** @type {CcSelectEvent} */ event) => this._onToggleChange(feature, event.detail)}
+          ></cc-toggle>
+        </div>
+        <div>${feature.description}</div>
+        ${feature.documentationLink != null || feature.feedbackLink != null
+          ? html`
+              <div class="footer">
+                ${feature.documentationLink != null
+                  ? html`
+                      <cc-link
+                        .href=${feature.documentationLink}
+                        a11y-desc="${i18n('cc-feature-list.documentation-link')} - ${feature.name}"
+                        .icon=${iconInfo}
+                      >
+                        ${i18n('cc-feature-list.documentation-link')}
+                      </cc-link>
+                    `
+                  : ''}
+                ${feature.feedbackLink != null
+                  ? html`
+                      <cc-link
+                        .href=${feature.feedbackLink}
+                        a11y-desc="${i18n('cc-feature-list.feedback-link')} - ${feature.name}"
+                        .icon=${iconMessage}
+                      >
+                        ${i18n('cc-feature-list.feedback-link')}
+                      </cc-link>
+                    `
+                  : ''}
+              </div>
+            `
+          : ''}
+      </div>
+    `;
+  }
+
+  render() {
+    return html`
+      <cc-block>
+        <div slot="header-title">${i18n('cc-feature-list.title')}</div>
+        <div slot="content">
+          ${this.state.type === 'loaded' && this.state.featureList.length > 0
+            ? html`<cc-notice message=${i18n('cc-feature-list.notice')}></cc-notice>`
+            : ''}
+          <div class="feature-list">
+            ${this.state.type === 'loading' ? html` <cc-loader></cc-loader>` : ''}
+            ${this.state.type === 'error'
+              ? html` <cc-notice intent="warning" message=${i18n('cc-feature-list.error')}></cc-notice> `
+              : ''}
+            ${this.state.type === 'loaded'
+              ? this.state.featureList.length > 0
+                ? this.state.featureList.map((feature) => this._renderFeature(feature))
+                : html` <cc-notice intent="info" message=${i18n('cc-feature-list.no-data')}></cc-notice> `
+              : ''}
```

Comment:
nitpick: we usually avoid to have mutliple nested conditional operators and divide the two conditions into two mustaches.
suggestion: 

```js
// We'd put this before the return of the render
// Note: maybe something like isFeatureListEmpty would be better, I let you decide the naming
const empty = this.state.featureList.length === 0;
```

```suggestion    
            ${this.state.type === 'loaded' && empty
                ? html` <cc-notice intent="info" message=${i18n('cc-feature-list.no-data')}></cc-notice> `
              : ''}
            ${this.state.type === 'loaded' && !empty
                ? this.state.featureList.map((feature) => this._renderFeature(feature))
             : ''}      
```